README.Rmd

```diff
@@ -28,7 +28,7 @@ library(ggplot2)
 library(knitr)
 ```
 
-The `scoringutils` package provides a collection of metrics and proper scoring rules and aims to make it simple to score probabilistic forecasts against the true observed values. 
+The `scoringutils` package provides a collection of metrics and proper scoring rules and aims to make it simple to score probabilistic forecasts against the observed values. 
```

Comment:
does it need to be the observed values vs observed values?

Reply:
Good point, fixed.